doc/source/user_guide/io.rst

```diff
@@ -137,7 +128,9 @@ usecols : list-like or callable, default ``None``
 
   .. ipython:: python
 
-     from io import StringIO, BytesIO
+     import pandas as pd
+     pd.options.display.max_rows = 15
```

Comment:
I think you can actually remove this line. 

I quickly checked and there are only a few longer dataframes shown in this page, and they are all longer. So with the current default of only showing first/last 5 rows when truncated, this option has no effect (it would only have the effect that dataframes longer than 15 but shorter than 60 would still be truncated, but such dataframes are not present in this file)

Reply:
Made the changes in commit d3036f3.